CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v0.16b1...HEAD)
 
+- Remove Configuration
+  ([#1523](https://github.com/open-telemetry/opentelemetry-python/pull/1523))
```

Comment:
Please move this under the `Removed` heading

Reply:
Done!